ui/store/actions.ts

```diff
@@ -4741,6 +4741,16 @@ export function setUseExternalNameSources(val: boolean): void {
   }
 }
 
+///: BEGIN:ONLY_INCLUDE_IF(transaction-simulation)
+export function setUseSimulations(val: boolean): void {
```

Comment:
Again for clarity, should we rename all the various methods and actions to `TransactionSimulations`?

Reply:
Renamed in https://github.com/MetaMask/metamask-extension/pull/23604/commits/c775aae0988b548cfd88d771e552801270597613